sampo/scheduler/genetic/operators.py

```diff
@@ -235,7 +247,7 @@ def is_chromosome_correct(chromosome: ChromosomeType,
     :return:
     """
     return is_chromosome_order_correct(chromosome, parents) and \
-           is_chromosome_contractors_correct(chromosome, node_indices)
+        is_chromosome_contractors_correct(chromosome, node_indices)
```

Comment:
the same, alignment... i suppose that old alignment is better to read, isn't it?